### DISCUSSION TITLE
Feature: Backup / Restore Mechanism for Chatwoot Installations

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#8191] [Feature Requests]
```[tasklist]
### Tasks
- [ ] Add an option in super admin pannel to generate database backups into storage
- [ ] https://github.com/chatwoot/docs/issues/428
- [ ] Ability to generate backup via  cwctl
- [ ] Ability to restore an installation from Chatwoot back ups
```

**Comment by nithindavid** [comment]
@vishnu-narayanan Can you please have a look at this and see if this is still relevant or plans on this?

**Comment by rihards-simanovics** [comment]
There should also be a consideration for creating backups using CLI command (for automation) or an ability to use curl or wget to securly download backups using the web protocols (http|https).

In the mean time is there a crude but a way to create backups, meaning specifc folders, files (of course DBs) etc?

**Comment by pranavrajs** [comment]
Moving this to feature discussion as the feature is not planned in the immediate roadmap.
